infra/gcp/ensure-staging-storage.sh

```diff
@@ -318,6 +318,11 @@ color 6 "Configuring special case for k8s-staging-releng-test"
 color 6 "Configuring special case for k8s-staging-cluster-api-gcp"
 (
     readonly STAGING_PROJECT="k8s-staging-cluster-api-gcp"
+    readonly serviceaccount="$(svc_acct_email "${STAGING_PROJECT}" "gcb-builder-cluster-api-gcp")"
+
     enable_api "${STAGING_PROJECT}" compute.googleapis.com
+    ensure_project_role_binding "${STAGING_PROJECT}" "serviceAccount:${serviceaccount}" "roles/compute.instanceAdmin.v1"
+    ensure_project_role_binding "${STAGING_PROJECT}" "serviceAccount:${serviceaccount}" "roles/iam.serviceAccountUser"
+    ensure_project_role_binding "${STAGING_PROJECT}" "serviceAccount:${serviceaccount}" "roles/editor"
```

Comment:
This is not needed for packer images builds. This role allow permissions on services outside of the scope of `compute.googleapis.com` service.